.docker/app/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:11.15.0-alpine
+
+WORKDIR /app
+RUN chown 1000:1000 /app
+
+USER 1000
+
+COPY --chown=1000:1000 package.json package.json
+COPY --chown=1000:1000 yarn.lock yarn.lock
+
+RUN yarn install
+
+COPY --chown=1000:1000 .env.dist .env
```

Comment:
you can use a single COPY : `COPY --chown=1000:1000 . /app`
and declare a `.dockerignore` file where you allow the files/dirs that should be copied ;)

Reply:
Thanks for that advice, cleaner indeed !